x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_flights/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "sample_data_flights",
+  "title": "Kibana sample data flights",
+  "description": "Find anomalies in mean flight delays",
```

Comment:
For consistency with the job description, I'd drop the plural and change to `Find anomalies in mean flight delay`

Reply:
Fixed!